### PR TITLE
Update GitHub API library. Add proxy support.

### PR DESCRIPTION
**Pull request by ragrus-nbl**
Issue #41 fixed

### CODE REVIEW

File: src/Readers/GithubReader.js [hunk 1]
```diff
@@ -131,17 +132,22 @@ class GithubReader extends AbstractReader {
...
   static fetch(source, username, password) {
-
+    var agent = null;
+    if (process.env.HTTPS_PROXY) {
+      agent = HttpsProxyAgent(process.env.HTTPS_PROXY);
+    } else if (process.env.https_proxy) {
+      agent = HttpsProxyAgent(process.env.https_proxy);
+    }
     const github = new GitHubApi({
       version: '3.0.0',
```
Comment (ppetrosh) [inline src/Readers/GithubReader.js]: Should the version be updated?

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
   "homepage": "https://github.com/electricimp/Builder#readme",
```
Comment (ppetrosh) [inline package.json]: Please bump up the Builder version up to 2.4.0

Comment (ragrus-nbl) [reply package.json]: Updated

File: src/Readers/GithubReader.js (hunk 1 above)
Comment (ragrus-nbl) [reply src/Readers/GithubReader.js]: Current version of GitHub API lib doesn't require this parameter, so I removed it.

**Pull request merged**